### PR TITLE
Plugins in the ecosystem page were divided into core and  community

### PR DESCRIPTION
**Pull request by srmarjani**

### CODE REVIEW

**Review by mcollina — COMMENTED** [review]
Good work! Would you mind including a screenshot of the new page?

**Comment by srmarjani** [comment]
I am at work right now.
I will send you tonight.

**Comment by lmammino** [comment]
Great Job @srmarjani,

Just pulled off your branch and created a screenshot (see at the end of this comment)!

One thing that I would love to have is some sort of summary/navigation (with links) at the top. There's a lot to scroll and it's not obvious there are two sections.

@mcollina, @delvedor, what do you guys think? We could also get this one merged first and figure out the navigation later.


127 0 0 1_8080_ecosystem_(iPad Pro)

**Comment by cemremengu** [comment]
@lmammino I think a tabbed view would be better

File: src/website/layouts/ecosystem.html [hunk 1]
```diff
@@ -13,11 +13,30 @@ <h1 class="title">
 <div class="container">
   <section class="section">
     <div class="content">
-      <h2>Currently, there are <strong>{{ data.ecosystem.plugins | length }}</strong> officially recognized plugins</h2>
+      <h2>Currently, there are <strong>{{ data.ecosystem.plugins.corePlugins | length }}</strong> officially recognized plugins</h2>
```
Comment (mcollina) [inline src/website/layouts/ecosystem.html]:
```suggestion
      <h2>Currently, there are <strong>{{ data.ecosystem.plugins.corePlugins | length }}</strong> core plugins</h2>
```

File: src/website/layouts/ecosystem.html [hunk 2]
```diff
@@ -13,11 +13,30 @@ <h1 class="title">
 <div class="container">
   <section class="section">
     <div class="content">
```
Comment (mcollina) [inline src/website/layouts/ecosystem.html:15]: I would add an index on top that tells:

1. there are X core plugins and Y community plugins
1. a core plugin is a plugin maintained by the fastify team, and we make our best to maintain them according to https://github.com/fastify/fastify/blob/master/docs/LTS.md.
1. we guarantee that every community plugin respects Fastify best practices (tests, etc) at the time they have been added to the list. We offer no guarantee on their maintenance.

cc @fastify/fastify

Comment (srmarjani) [reply src/website/layouts/ecosystem.html:15]: @mcollina 
I must do anything? 
I must add them or not?

Comment (mcollina) [reply src/website/layouts/ecosystem.html:15]: Add those in a way that fits well in the page.

**Comment by srmarjani** [comment]
Hi 
I made change that @mcollina suggested.
this is a screenshot.

image

**Review by mcollina — APPROVED** [review]
LGTM

**Comment by lmammino** [comment]
I like it :)

**Review by lmammino — APPROVED** [review]

**Pull request merged**